clients/vscode/src/chat/ChatViewProvider.ts

```diff
@@ -128,6 +130,50 @@ export class ChatViewProvider implements WebviewViewProvider {
 
         this.sendMessage(chatMessage);
       },
+      onApplyInEditor: (content: string) => {
+        const editor = window.activeTextEditor;
+        if (editor) {
+          const document = editor.document;
+          const selection = editor.selection;
+          const text = editor.document.getText(selection);
+          const { start } = selection;
+
+          // Find the minimum indent
+          const lineText = document.lineAt(start.line).text;
+          const selectedLines = text.split("\n");
+          const allLines = [lineText].concat(selectedLines.slice(1));
+          let subsequentLinesMinIndent = "";
+          let subsequentLinesMinIndentLength = Infinity;
+          allLines.forEach((line) => {
```

Comment:
I suggest directly using the indent of the current line, which should be sufficient, the current line refers to the line at the `selection.start` position. There is no need to traverse through the subsequent lines of the selection to find the shortest indent.

Reply:
update to only detect the indent of the first line